### PR TITLE
add i18n to Snappass

### PR DESCRIPTION
**Pull request by systeembeheerder**
I am not a (python) programmer. I just tried to add i18n to snappass because I get tired from editing after each update. 

Please take a look and critique this branch. I guess the compilation of the .mo files should be some part of the build process?

This would solve https://github.com/pinterest/snappass/issues/118

### CODE REVIEW

File: snappass/main.py [hunk 1]
```diff
@@ -5,18 +5,18 @@
 import redis
 
 from cryptography.fernet import Fernet
-from flask import abort, Flask, render_template, request, jsonify
+from flask import abort, Flask, render_template, request, jsonify, g
```
Comment (xia0pin9) [inline snappass/main.py:8]: seems no need to import `g`

File: snappass/main.py [hunk 2]
```diff
@@ -25,6 +25,11 @@
 app.config.update(
     dict(STATIC_URL=os.environ.get('STATIC_URL', 'static')))
 
+# Set up Babel
+def get_locale():
+    return request.accept_languages.best_match(['en', 'nl'])
```
Comment (xia0pin9) [inline snappass/main.py:30]: nit: maybe also add some other supported languages?

**Review by xia0pin9 — APPROVED** [review]

File: snappass/templates/password.html [hunk 3]
```diff
@@ -3,22 +3,22 @@
...
-        <button title="Copy to clipboard" type="button" class="btn btn-primary copy-clipboard-btn"
+        <button title="{{ _('Copy to clipboard') }}" type="button" class="btn btn-primary copy-clipboard-btn"
               id="copy-clipboard-btn" data-clipboard-target="#password-text"
               data-placement='bottom'>
           <i class="fa fa-clipboard"></i>
         </button>
       </div>
     </div>
-    <p>The secret has now been permanently deleted from the system, and the URL will no longer work. Refresh this page to verify.</p>
+    <p>{{ _('The secret has now been permanently deleted from the system, and the URL will no longer work. Refresh this page to verify.' }}</p>
```
Comment (xia0pin9) [inline snappass/templates/password.html:21]: seems missing an ending `)`

**Review by xia0pin9 — CHANGES_REQUESTED** [review]

File: snappass/templates/password.html (hunk 3 above)
Comment (systeembeheerder) [reply snappass/templates/password.html]: thanx. fixed

**Review by xia0pin9 — APPROVED** [review]

File: snappass/main.py [hunk 4]
```diff
@@ -56,7 +61,6 @@ def inner(*args, **kwargs):
                 return abort(500)
     return inner
 
-
```
Comment (xia0pin9) [inline snappass/main.py:59]: the extra empty lines are needed: 
```
  flake8: commands[1]> flake8
  ./snappass/main.py:29:1: E302 expected 2 blank lines, found 1
  def get_locale():
  ^
  ./snappass/main.py:31:1: E305 expected 2 blank lines after class or function definition, found 0
  babel = Babel(app, locale_selector=get_locale)
  ^
  ./snappass/main.py:64:1: E302 expected 2 blank lines, found 1
  def encrypt(password):
  ^
  ./snappass/main.py:219:1: E305 expected 2 blank lines after class or function definition, found 1
  if __name__ == '__main__':
  ^
  flake8: exit 1 (0.38 seconds) /home/runner/work/snappass/snappass> flake8 pid=1963
.pkg: _exit> python /opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/pyproject_api/_backend.py True setuptools.build_meta __legacy__
  flake8: FAIL code 1 (11.56=setup[7.48]+cmd[3.70,0.38] seconds)
  evaluation failed :( (12.82 seconds)
```

**Review by xia0pin9 — CHANGES_REQUESTED** [review]

File: snappass/main.py [hunk 5]
```diff
@@ -25,6 +26,13 @@
 app.config.update(
     dict(STATIC_URL=os.environ.get('STATIC_URL', 'static')))
 
+# Set up Babel
+def get_locale():
```
Comment (xia0pin9) [inline snappass/main.py:30]: also need another empty line here:
```
  flake8: commands[1]> flake8
  ./snappass/main.py:30:1: E302 expected 2 blank lines, found 1
  def get_locale():
  ^
  flake8: exit 1 (0.57 seconds) /home/runner/work/snappass/snappass> flake8 pid=1916
```

**Review by xia0pin9 — CHANGES_REQUESTED** [review]

File: snappass/main.py (hunk 5 above)
Comment (systeembeheerder) [reply snappass/main.py:31]: thanx for your patience; i'm learning a lot.

**Review by xia0pin9 — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**